modules/map/relative_map/proto/navigation.proto

```diff
@@ -38,4 +38,7 @@ message MapMsg {
 
   // localization
   optional apollo.localization.LocalizationEstimate localization = 5;
+
+  // the lane id of the navigation path which the vehicle is currently on
+  optional string vechicle_lane_id = 6;
```

Comment:
typo: vechicle > vehicle 

it is better to decouple the vehicle state with map information.

Reply:
Yes, your consideration makes sense, and I withdraw this submission.